chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_confirmation.html

```diff
@@ -1,13 +1,36 @@
 {% extends "admin/delete_confirmation.html" %}
-{% load amend prepend from poleno.amend %}
+{% load amend prepend before after delete from poleno.amend %}
 
 {% block content %}
   {% amend %}
     {{ block.super }}
+    {% after path="./ul[1]" %}
```

Comment:
V `path="./ul[1]` sa mi nepaci ta `[1]`. Nie je to rozbustne. Lepsie by bolo povedat "za posledny ul" namiesto "za prvy ul a dufame, ze je tam iba jeden ul".

Da sa to nejak? Ak nie, tak potom by som radej dal "pred form", nez "za prvy ul".

Reply:
:ok:

Reply:
:ok: